tests/test_xdmf.py

```diff
@@ -0,0 +1,225 @@
+import os
+import re
+
+import pytest
+from more_itertools import distinct_combinations
+from unyt import Unit, assert_allclose_units
+
+import yt
+from yt_idefix.api import PlutoXdmfDataset
+
+# A sample list of units for test.
+# The first three values are chosen randomly
+# and others are calculated correspondingly.
+SAMPLE_UNITS = {
+    "time_unit": (2.0, "s"),
+    "length_unit": (4.0, "cm"),
+    "mass_unit": (5.0, "kg"),
+    "density_unit": (0.078125, "kg/cm**3"),
+    "velocity_unit": (2.0, "cm/s"),
+    "magnetic_unit": (62.66570686577499, "gauss"),
+}
+
+
+def test_class_validation(xdmf_file):
+    file = xdmf_file
+    cls = {
+        "plutoXDMF": PlutoXdmfDataset,
+    }[file["kind"]]
+    assert cls._is_valid(file["path"])
+
+
+def test_parse_pluto_metadata(pluto_xdmf_file):
```

Comment:
many of these tests appear to be straight up copy/pasted from `test_vtk.py`.

I think copying *all* tests we already have is an overkill, so let's keep just the last 3, starting from `test_slice_plot`, to bring the diff back to a manageable size. I should be able to generalise most of our tests later without duplications.

Reply:
Done!